package.json

```
@@ -8,10 +8,13 @@
     "build:renderer": "cross-env NODE_ENV=production webpack --config ./.erb/configs/webpack.config.renderer.prod.babel.js",
     "rebuild": "electron-rebuild --parallel --types prod,dev,optional --module-dir src",
     "lint": "cross-env NODE_ENV=development eslint . --cache --ext .js,.jsx,.ts,.tsx",
-    "package": "rm -rf src/dist && yarn build && electron-builder build --publish never",
+    "clean": "node ./.erb/scripts/Clean.js dist release",
+    "clean:dist": "node ./.erb/scripts/Clean.js dist",
+    "clean:release": "node ./.erb/scripts/Clean.js release",
+    "package": "cross-env yarn clean:dist && yarn build && electron-builder build --publish never",
```

One of ERB's goals since the 2.0 release was to minimize the number of npm scripts. To abide by that principle we should remove the clean scripts here and inline the `clean:dist` script into `package`:

```suggestion
    "package": "rm -rf build/dist && yarn build && electron-builder build --publish never",
```